src/app/src/main/kotlin/org/imperial/mrc/hint/models/Response.kt

```diff
@@ -2,14 +2,16 @@ package org.imperial.mrc.hint.models
 
 import com.fasterxml.jackson.databind.ObjectMapper
 
-data class SuccessResponse(val data: Any){
+data class SuccessResponse(val data: Any?){
     val errors = mapOf<Any, Any>()
     val status = "success"
 }
 
 
 fun SuccessResponse.toJsonString() = ObjectMapper().writeValueAsString(this)
 
+val EmptySuccessResponse = SuccessResponse(null)
```

Comment:
Null won't work so well for the front-end. Can we use `true` or the string "OK" instead?